app/build.gradle

```diff
@@ -113,8 +113,8 @@ dependencies {
 	implementation 'com.google.android:flexbox:0.3.1'
 	implementation 'com.mikepenz:materialdrawer:6.0.0'
 	implementation 'com.mikepenz:aboutlibraries:6.0.0'
-	implementation 'com.mikepenz:fastadapter-commons:3.0.3@aar'
-	implementation 'com.mikepenz:fastadapter-extensions-expandable:3.0.3@aar'
```

Comment:
Would you mind upgrading this to the latest `3.2.0`? The CI fails for `3.0.5` for some reason.

Reply:
Yeah, no problem. 😄  It's already at `3.2.2` 🙊

Reply:
*`3.2.3` 😆